Get the real filesize

kdruelle opened the pull request:
In case the file is a symlink, get the size from the real file not the symlink itself

DBezemer commented:
I think by itself this shell_exec is not the right way to do it. PHP has a built-in filesize() function that works across platforms, so better to use that.

kdruelle commented:
I agree. but at the top of this function I noticed this comment : 

>  Since filesize() was giving trouble with files larger
>  than 2gb, I looked for a solution and found this great
>  function by Alessandro Marinuzzi from www.alecos.it on
>  http://stackoverflow.com/questions/5501451/php-x86-how-
>  to-get-filesize-of-2gb-file-without-external-program
>  I changed the name of the function and split it in 2,
>  because I do not want to display it directly.

ignacionelson commented:
Thanks for the code!

kdruelle commented:
You're welcome
